src/main/java/tdsql/tester/TestParser.java

```diff
@@ -0,0 +1,5 @@
+package tdsql.tester;
+
+public class TestParser {
```

Comment:
Ignore this, this is the next steps after this PR to implement a parser that parses these files into some sort of Java representation of a test suite.